### PR TITLE
Feature/configuration

### PR DESCRIPTION
**Pull request by xlasserre**
### Boilerplate for the target app

The following dependencies were added and configured:

* Redux
* React native navigation
* Redux-thunk
* ESLint
* CircleCI

### CODE REVIEW

File: .eslintrc.js [hunk 1]
```diff
@@ -0,0 +1,84 @@
+// "off" or 0 - turn the rule off
```
Comment (bernabe9) [inline .eslintrc.js]: We have our own `.eslintrc.js` file (https://github.com/rootstrap/react-native-base/blob/master/.eslintrc)

I would prefer if you use it to get used to our code style. Anyways, we can discuss other rules to add or remove :)

File: src/components/App.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+import { Navigation } from 'react-native-navigation';
+import { Provider } from 'react-redux';
+import registerScreens from '../screens';
+import configureStore from '../store/configureStore';
+
+const store = configureStore();
+registerScreens(store, Provider);
+
+class App extends React.Component {
+  constructor(props) {
```
Comment (bernabe9) [inline src/components/App.js]: `props` isn't necessary

File: src/components/App.js [hunk 3]
```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { Navigation } from 'react-native-navigation';
+import { Provider } from 'react-redux';
+import registerScreens from '../screens';
+import configureStore from '../store/configureStore';
+
+const store = configureStore();
+registerScreens(store, Provider);
+
+class App extends React.Component {
```
Comment (bernabe9) [inline src/components/App.js]: I think that you don't need to extend this class to a React Component, just a class should work

File: src/containers/LoginScreen/index.js [hunk 4]
```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import {
```
Comment (bernabe9) [inline src/containers/LoginScreen/index.js]: Write inline when the line doesnt exceed 80 characters

File: src/containers/LoginScreen/styles.js [hunk 5]
```diff
@@ -0,0 +1,9 @@
+import { StyleSheet } from 'react-native';
+
+const styles = StyleSheet.create({
+  color: {
+    backgroundColor: '#EA0000',
```
Comment (bernabe9) [inline src/containers/LoginScreen/styles.js]: Please create a styles constants file and store this color in it.

**Review by bernabe9 — CHANGES_REQUESTED** [review]
Excellent PR!

File: .eslintrc.js (hunk 1 above)
Comment (xlasserre) [reply .eslintrc.js]: Sure, I can use that!

File: src/components/App.js (hunk 3 above)
Comment (xlasserre) [reply src/components/App.js]: Yeah, you are right, props and React.Component is not needed here.

File: src/containers/LoginScreen/styles.js (hunk 5 above)
Comment (xlasserre) [reply src/containers/LoginScreen/styles.js]: Sure! I just had this to make sure I was showing the screen and everything is not blank, but I could do that now and leave the styles constants file ready to be used later.

**Review by bernabe9 — APPROVED** [review]

**Review by matiasmansilla1989 — APPROVED** [review]

**Pull request merged**